tools/ola_trigger/contrib/hue_trigger.conf

```diff
@@ -0,0 +1,47 @@
+# OLA Trigger config to control Philips Hue Lux lights
```

Comment:
It may also be worth renaming the file to something like philips_hue_lux.conf, and dropping the trigger bit, given it's already in the ola_trigger folder now.